Tests/TestingMacrosTests/TestDeclarationMacroTests.swift

```diff
@@ -325,8 +325,6 @@ struct TestDeclarationMacroTests {
       ("@Test @_unavailableFromAsync @MainActor func f() {}", nil, "MainActor.run"),
       ("@Test @available(*, noasync) func f() {}", nil, "__requiringTry"),
       ("@Test @_unavailableFromAsync func f() {}", nil, "__requiringTry"),
-      ("@Test(arguments: []) func f(i: borrowing Int) {}", nil, "copy"),
-      ("@Test(arguments: []) func f(_ i: borrowing Int) {}", nil, "copy"),
```

Comment:
Are these removals because we're no longer supporting `borrowing` parameters?

Reply:
No, it's because we no longer need to copy the arguments (which is what these cases were looking for.)